lib/thermex/monitor.ex

```diff
@@ -31,29 +36,14 @@ defmodule Thermex.Monitor do
   end
 
   defp read_temperature(filename) do
-    lines = filename
-    |> File.stream!
-    |> Stream.map(&String.strip/1)
-    |> Enum.take(2)
-
-    {temperature, _} = parse_temperature(List.first(lines), List.last(lines))
-    temperature / 1000
-  end
-
-  defp parse_temperature(first_string, second_string) do
-    cond do
-      String.ends_with?(first_string, "YES") ->
-        String.split(second_string, "=")
-        |> List.last
-        |> Integer.parse
-      true ->
-        nil
-    end
+    filename
+    |> File.read!
```

Comment:
The main reason I changed it was because I wanted to write some unit tests for parsing the contents and it was easier to write the test as passing in a string rather than a stream.